docker/cron/adpay-payment.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ ! -e adpay_export_in_progress ]; then
+    touch adpay_export_in_progress
+
+    ./artisan ops:adpay:campaign:export
+    ./artisan ops:adpay:event:export
```

Comment:
What is the difference between `adpay-payment.sh` and `adpay-export.sh`?

Reply:
Right. Fixed.